openmc/tallies.py

```diff
@@ -21,9 +22,58 @@ def reset_auto_tally_id():
 
 
 class Tally(object):
+    """A tally defined by a set of scores that are accumulated for a list of
+    nuclides given a set of filters.
+
+    Parameters
+    ----------
+    tally_id : int
```

Comment:
I see that the id is autogenerated in the property setter, but I don't see where the name is autogenerated. Is it part of the Tally class or elsewhere?